runtime/doc/filetype.txt

```diff
@@ -554,6 +551,18 @@ Variables:
                                :let b:man_default_sections = '3,2'
 *g:man_hardwrap*            Hard-wrap to $MANWIDTH. May improve layout.
 
+To use Nvim as a manpager: >
+     export MANPAGER='nvim +Man!'
+
+Note that when running `man` from the shell and with that `MANPAGER` in your
+environment, `man` will pre-format the manpage using `groff`. Thus, Neovim
+will inevitably display the manual page as it was passed to it from stdin. One
+of the caveats of this is that the width will _always_ be hard-wrapped and not
+soft wrapped as with `:Man`. You can set in your environment: >
+     export MANWIDTH=999
+
+So `groff`'s pre-formatting output will be the same as with `:Man` i.e soft-wrapped.
+
```

Comment:
According to `man(1)` `$COLUMNS` also should be taken into account?!
```
       MANWIDTH
              If  $MANWIDTH  is set, its value is used as the line length for which manual
              pages should be formatted.  If it is not set, manual pages will be formatted
              with  a  line length appropriate to the current terminal (using the value of
              $COLUMNS, an ioctl(2) if available, or falling back to 80 characters if nei‐
              ther  is  available).  Cat pages will only be saved when the default format‐
              ting can be used, that is when the terminal line length is between 66 and 80
              characters.
```
(man-db 2.8.6.1-1)

Reply:
..but that then still hardwraps - I see.

Reply:
Maybe it should be worth mentioning that internally, man.vim uses `MANPAGER=cat` and `MANWIDTH=999` to get the contents of the manual page?